src/scripts/pokemons/PokemonList.ts

```diff
@@ -17769,6 +18668,23 @@ const pokemonList = createPokemonArray(
             'specialDefense': 100,
             'speed': 115,
         },
+    },
+    {
+        'id': -793,
+        'name': '???',
+        'type': [PokemonType.Rock, PokemonType.Poison],
+        'eggCycles': 120,
+        'levelType': LevelType.slow,
+        'exp': 257,
+        'catchRate': 45,
+        'base': {
+            'hitpoints': 109,
+            'attack': 53,
+            'specialAttack': 127,
+            'defense': 47,
+            'specialDefense': 131,
+            'speed': 103,
+        },
```

Comment:
What is this Pokémon supposed to be?

Reply:
??? is my attempt at recreating the Nihilego fight that happens during the first visit to the Aether Paradise (where it is also named ??? in the main series). It's a gym battle so no catching, and it's a negative ID because I also don't want it ever appearing in the dex. Which means this depends on negative IDs being reverted to their old state and current negative event mons being turned into decimals.

Reply:
`gymList['Aether Foundation'] = new Gym(
    'Ultra Wormhole',
    'Aether Foundation',
    [new GymPokemon('???', 2458300, 27)],
    BadgeEnums.Elite_Nihilego,
    128000,
    'The creature escaped back into the ultra wormhole.',
    [new GymBadgeRequirement(BadgeEnums.AkalaKahuna)]
);`